Makefile

```diff
@@ -15,16 +17,21 @@ sd-branch: v8
 		git checkout sd-master && \
 		${DEPOT_PATH} gclient sync
 
-v8/sd-build.sh: sd-branch
+v8/${BUILD_SCRIPT}: sd-branch
 	@echo "#!/bin/sh" > $@
 	@echo "${DEPOT_PATH} tools/dev/v8gen.py x64.release" >> $@
 	@echo "${DEPOT_PATH} ninja -C out.gn/x64.release d8" >> $@
 	@chmod +x $@
 	@echo "\n\n-------- soft-dev specific info --------"
-	@echo "\nTo build d8 enter the 'v8' dir and run $@"
+	@echo "\nTo build d8 enter the 'v8' dir and run ${BUILD_SCRIPT}"
 
-v8/sd-run-bench.sh: sd-branch
-	@echo "#!/bin/sh" > $@
-	@echo "tools/run_perf.py --arch x64 --binary-override-path out.gn/x64.release/d8 --buildbot test/js-perf-test/RegExp.json" >> $@
-	@chmod +x $@
-	@echo "\nTo run the example benchmark, enter the 'v8' dir and run $@"
+v8/${RUN_SCRIPT}: sd-branch
+	@cp scripts/${RUN_SCRIPT} $@
+	@chmod +x v8/${RUN_SCRIPT}
+	@echo "To run the example benchmark, cd to the v8 dir and run ${RUN_SCRIPT}"
```

Comment:
Shall we mention the files to edit to change the pexec and iter counts?

Reply:
Pushed a commit for this.